### ISSUE TITLE
is there any way to get the layered architecture of the facemesh graph model?

### ISSUE THREAD

**Issue by ThejanW** [tensorflow/tfjs#3356]
To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### Code to reproduce the bug / link to feature request
If you would like to get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

GitHub issues for this repository are tracked in the [tfjs union repository](https://github.com/tensorflow/tfjs/issues).

Please file your issue there, following the guidance in [that issue template](https://github.com/tensorflow/tfjs/blob/master/ISSUE_TEMPLATE.md).

**Comment by ThejanW** [comment]
Facemesh model is in graph format https://tfhub.dev/mediapipe/tfjs-model/facemesh/1/default/1 
I'd like to see its model architecture. There's nothing much I could find even in the published paper. In the model card https://drive.google.com/file/d/1VFC_wIpw4O7xBOiTgUldl79d9LA-LsnA/view it says "​Convolutional Neural Network: MobileNetV2-like with customized blocks for real-time performance.". Authors have revealed neither the customizations they've done nor the model architecture. Any resources on this would be much appreciated.

**Comment by annxingyuan** [comment]
Hi @ThejanW - here is a link to a JSON object describing the model's topology: https://storage.googleapis.com/tfhub-tfjs-modules/mediapipe/tfjs-model/facemesh/1/default/1/model.json

Hopefully this helps! These objects are available for all our graph models - if you open the network tab you'll typically see a "model.json" link.

**Comment by ThejanW** [comment]
@annxingyuan I humbly appreciate your prompt response. I played around the provided graph model and even converted that to TesorFlow python SavedModel format. I've already looked into the provided model.json. It's apparent that this model topology file is generated using tfjs converter from a TensorFlow SavedModel. 

Will you be exposing the TensorFlow model definition scripts or even a tfjs layers model in the near future. From a layers model, what I meant was loading a model.json file with a command like this `const model = await tf.loadLayersModel(‘path/to/model.json’);`

My goal is to customize this architecture for a different research. But I have to first see the model architecture as layers rather than as ops and nodes to perform my customizations. 

Thanks.
